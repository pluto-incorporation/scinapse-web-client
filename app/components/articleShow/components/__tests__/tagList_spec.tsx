jest.unmock("../tagList");

import * as React from "react";
import TagList, { ITagListProps } from "../tagList";
import { shallow, ShallowWrapper } from "enzyme";

describe("<TagList /> component", () => {
  let tagListWrapper: ShallowWrapper<ITagListProps>;
  const mockTags = ["Open Access Paper", "Tylor Shin Awesome"];

  beforeEach(() => {
    tagListWrapper = shallow(<TagList tags={mockTags} />);
  });

  it("should match snapshot", () => {
    expect(tagListWrapper.html()).toMatchSnapshot();
  });
});
