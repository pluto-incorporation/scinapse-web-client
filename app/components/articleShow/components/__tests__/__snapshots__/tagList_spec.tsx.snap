// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TagList /> component should match snapshot 1`] = `"<div class=\\"tagListWrapper\\"><a class=\\"tagItem\\">Open Access Paper</a><a class=\\"tagItem\\">Tylor Shin Awesome</a></div>"`;
